attributes-kt/src/commonMain/kotlin/space/kscience/attributes/Attributes.kt

```diff
@@ -0,0 +1,129 @@
+/*
+ * Copyright 2018-2023 KMath contributors.
+ * Use of this source code is governed by the Apache 2.0 license that can be found in the license/LICENSE.txt file.
+ */
+
+package space.kscience.attributes
+
+import kotlin.jvm.JvmInline
+
+/**
+ * A set of attributes. The implementation must guarantee that [content] keys correspond to its value types.
```

Comment:
Maybe, it is better like this, isn't it?
```suggestion
 * A set of attributes. The implementation must guarantee that [content] keys correspond to their value types.
```